packages/vite/src/node/server/sourcemap.ts

```diff
@@ -11,6 +11,7 @@ export async function injectSourcesContent(
   map.sourcesContent = []
   await Promise.all(
     map.sources.map(async (sourcePath, i) => {
+      if (!sourcePath) return
```

Comment:
**suggestion (dx):** I know it would be more code, but also it is more readable

Filter sources beforehand and then call `map`
```js
    map.sources
       .filter((sourcePath) => !!sourcePath)
       .map(async (sourcePath, i) => {
```